src/LibTemplateCMake/CMakeLists.txt

```diff
@@ -2,7 +2,12 @@
 # The actual file name of the library built is constructed based on conventions
 # of the native platform, such as lib<LIBRARY_TARGET_NAME>.a,
 # lib<LIBRARY_TARGET_NAME>.dylib or <LIBRARY_TARGET_NAME>.lib.
-set(LIBRARY_TARGET_NAME LibTemplateCMake)
+#
+# In order to use a library name that is not the same as the `EXPORT` (in the
+# `install` command), it is necessary to pass the `FIRST_TARGET` option to
+# `install_basic_package_files` in the main CMakeLists.txt file.
+# In this case the project name is used for both, therefore it is not necessary.
+set(LIBRARY_TARGET_NAME ${PROJECT_NAME})
```

Comment:
Thanks for this nice comment. I'm wondering whether it would be more suitable for a tutorial to explicitly use `FIRST_TARGET` (which I agree would not be necessary here, but is for the sake of teaching 😄) and then write that in this case it would have been sufficient to use the same `PROJECT_NAME` without `FIRST_TARGET`.
Either way, I'm fine so it is up to you to choose 😄

Reply:
I considered that as well, but I think in this case it is not really useful, it does not teach you anything more than what is explained in the comment. Also if you need to do something more complicated, you can easily read the documentation of the modules...